iceoryx_examples/ice_multi_publisher/README.md

```diff
@@ -1,5 +1,262 @@
-# Multiple publishers on the same topic
+# Multiple publishers for the same topic
+
+## Introduction
+A common use case is that often we have multiple sensors sending data of the same type, e.g. LIDAR data, and a subscriber is interested in the data of all those sensors. 
+
+This example shows how you can run multiple publishers and let them pblish of the same topic. 
+Here we only cover the specifics for this use case, for basic sending and receiving see the 
+ [icedelivery](../icedelivery/README.md) example. The example also demonstrates how you can change the maximum number of samples available at a subscriber when you resubscribe.
+## Run ice_multi_publisher
+
+Create four terminals and run one command in each of them. 
+
+    # If installed and available in PATH environment variable
+    iox-roudi
+    # If build from scratch with script in tools
+    $ICEORYX_ROOT/build/posh/iox-roudi
+
+    ./build/iceoryx_examples/ice_multi_publisher/iox-multi-publisher
+
+    ./build/iceoryx_examples/ice_multi_publisher/iox-subscriber
+
+    ./build/iceoryx_examples/ice_multi_publisher/iox-resubscriber
+
+
+This starts two publishers in the iox-multi-publisher application and 
+a subscriber in each of the other applications. The two publishers publish publish two independent counters. The data of bboth is made distinct by using an id, which has the values 1 and 2 respectively. 
+## Expected output
+
+The counters can differ depending on startup of the applications.
+
+### RouDi application
+
+    Reserving 84783040 bytes in the shared memory [/iceoryx_mgmt]
+    [ Reserving shared memory successful ] 
+    Reserving 149655680 bytes in the shared memory [/apex]
+    [ Reserving shared memory successful ] 
+    RouDi is ready for clients
+
+### Publisher application
+
+    Counter Instance sending: id 1 counter 0
+    Counter Instance sending: id 2 counter 0
+    Counter Instance sending: id 1 counter 1
+    Counter Instance sending: id 2 counter 1
+    Counter Instance sending: id 1 counter 2
+    Counter Instance sending: id 1 counter 3
+    Counter Instance sending: id 2 counter 2
+    Counter Instance sending: id 1 counter 4
+    Counter Instance sending: id 1 counter 5
+    Counter Instance sending: id 2 counter 3
+    Counter Instance sending: id 1 counter 6
+### Subscriber application
+
+    Waiting for data ... 
+    Received: id 1 counter 1
+    Received: id 2 counter 1
+    Received: id 1 counter 2
+    Waiting for data ... 
+    Received: id 1 counter 3
+    Received: id 2 counter 2
+    Received: id 1 counter 4
+    Waiting for data ... 
+    Received: id 1 counter 5
+    Received: id 2 counter 3
+    Received: id 1 counter 6
```

Comment:
I'd replace this section with some nice [asciinema](https://asciinema.org/) recordings. This applies to all examples. Will record them once 0.9 went live.

Reply:
Ok, but should be done for 1.0 when we rework and consolidate the documentation. We probably want an issue or this.